PyRDF/RDataFrame.py

```diff
@@ -55,7 +81,7 @@ def __init__(self, *args):
                 args[i] = self._get_vector_from_list(args[i])
 
         try:
-            ROOT.ROOT.RDataFrame(*args)  # Check if the args are correct
```

Comment:
I would prefer to keep this comment, since it might not be clear for new users why we do it at this point if we claim to be lazy.

Reply:
ok